apiserver/pkg/scaling/scaling.go

```diff
@@ -2,27 +2,41 @@ package scaling
 
 import (
 	"math"
+	"math/rand"
 	"strconv"
+	"time"
 
 	runnerv1alpha1 "github.com/devjoes/github-runner-autoscaler/operator/api/v1alpha1"
 	"k8s.io/klog/v2"
 )
 
 type Scaling struct {
-	MinWorkers  int32   `json:"minWorkers"`
-	MaxWorkers  int32   `json:"maxWorkers"`
-	ScaleFactor float64 `json:"scaleFactor"`
-	Linear      bool    `json:"linear"`
+	MinWorkers                int32   `json:"minWorkers"`
+	MaxWorkers                int32   `json:"maxWorkers"`
+	ScaleFactor               float64 `json:"scaleFactor"`
+	Linear                    bool    `json:"linear"`
+	ForceScaleUpWindowSecs    int32   `json:"forceScaleUpWindowSecs"`
```

Comment:
Perhaps consider using metav1.Duration types here?

Reply:
> i don't know about the code but i'd just like to check when the 20 day period is measured from, to ensure that we're not going to have all the runners everywhere scaling up simultaneously

It's 20 days + a random amount of time up to one day to avoid that exact problem